lib/experimental/PageLayouts/OverviewLayout/index.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { Dashboard } from "@/experimental/Widgets/Layout/Dashboard"
+import * as DashboardStories from "@/experimental/Widgets/Layout/Dashboard/index.stories"
+import { PageDecorator } from "@/lib/storybook-utils/pageDecorator"
+import { Placeholder } from "@/lib/storybook-utils/placeholder"
+import { ComponentProps } from "react"
+import { OverviewLayout } from "."
+import { DetailsItemsList } from "../Utils/DetailsItemsList"
+import * as DetailsItemsListStories from "../Utils/DetailsItemsList/index.stories"
+
+const DETAILS_ITEMS_ARGS = DetailsItemsListStories.default
+  .args as ComponentProps<typeof DetailsItemsList>
+
+const meta = {
+  component: OverviewLayout,
+  tags: ["autodocs"],
+  decorators: [PageDecorator],
+  args: {
+    children: <Placeholder className="h-[450px]">Main</Placeholder>,
+    sidepanel: {
+      title: DETAILS_ITEMS_ARGS.title,
+      items: DETAILS_ITEMS_ARGS.details,
+    },
```

Comment:
I'm worried that this won't scale well in the future, perhaps? But I guess it's easier to make an API less strict than more strict, so I'm not opposing the idea.

Reply:
Yep, maybe we end up having more cases, but I also thought it's easier to do that change in the future if we don't have this component too open and working in many different ways in our frontend